src/devices/src/virtio/block/event_handler.rs

```diff
@@ -85,10 +79,17 @@ impl Subscriber for Block {
     }
 
     fn interest_list(&self) -> Vec<EpollEvent> {
-        vec![EpollEvent::new(
-            EventSet::IN,
-            self.activate_evt.as_raw_fd() as u64,
-        )]
+        if self.is_activated() {
```

Comment:
I would add a comment here to tell the story of when this happens. FreshVM, Restored clone.

Reply:
done